### PR TITLE
CXF-90091 FCR to All CSP Connection

### PR DESCRIPTION
**Pull request by jkallem-equinix**
- FCR 2 Port,
- FCR 2  SP (Private)
- FCR 2 WAN
- FCR 2 AWS,
- FCR 2 Azure

### CODE REVIEW

**Comment by jkallem-equinix** [comment]
Screenshot 2024-04-09 at 11 03 51 PM

Test Result Screenshot

File: .github/workflows/terratests-prod-suite.yaml [hunk 1]
```diff
@@ -16,6 +16,12 @@ jobs:
       TEST_DATA_PROD_PORT_2_IBM2_CONNECTION: ${{ secrets.TEST_DATA_PROD_PORT_2_IBM2_CONNECTION }}
       TEST_DATA_PROD_PORT_2_PORT_CONNECTION: ${{ secrets.TEST_DATA_PROD_PORT_2_PORT_CONNECTION }}
       TEST_DATA_PROD_PORT_2_PRIVATE_SERVICE_PROFILE_CONNECTION: ${{ secrets.TEST_DATA_PROD_PORT_2_PRIVATE_SERVICE_PROFILE_CONNECTION }}
+      TEST_DATA_PROD_CLOUD_ROUTER_2_AWS_CONNECTION: ${{secrets.TEST_DATA_PROD_CLOUD_ROUTER_2_AWS_CONNECTION}}
+      TEST_DATA_PROD_CLOUD_ROUTER_2_AZURE_CONNECTION: ${{secrets.TEST_DATA_PROD_CLOUD_ROUTER_2_AZURE_CONNECTION}}
+      TEST_DATA_PROD_CLOUD_ROUTER_2_PORT_ROUTING_PROTOCOL_CONNECTION: ${{secrets.TEST_DATA_PROD_CLOUD_ROUTER_2_PORT_ROUTING_PROTOCOL_CONNECTION}}
+      TEST_DATA_PROD_CLOUD_ROUTER_2_SERVICE_PROFILE_CONNECTION: ${{secrets.TEST_DATA_PROD_CLOUD_ROUTER_2_SERVICE_PROFILE_CONNECTION}}
+      TEST_DATA_PROD_CLOUD_ROUTER_2_WAN_CONNECTION: ${{secrets.TEST_DATA_PROD_CLOUD_ROUTER_2_WAN_CONNECTION}}
+
```
Comment (srushti-patl) [inline .github/workflows/terratests-prod-suite.yaml:24]: nit: remove extra space

File: .github/workflows/terratests-prod-suite.yaml [hunk 2]
```diff
@@ -44,6 +50,13 @@ jobs:
...
           echo $TEST_DATA_PROD_PORT_2_PORT_CONNECTION >> "./examples/port-2-port-connection/terraform.tfvars.json"
           echo $TEST_DATA_PROD_PORT_2_PRIVATE_SERVICE_PROFILE_CONNECTION >> "./examples/port-2-private-service-profile-connection/terraform.tfvars.json"
+          
+          echo $TEST_DATA_PROD_CLOUD_ROUTER_2_AWS_CONNECTION >> "./tests/examples-without-external-providers/cloud-router-2-aws-connection/terraform.tfvars.json"
+          echo $TEST_DATA_PROD_CLOUD_ROUTER_2_AZURE_CONNECTION >> "./tests/examples-without-external-providers/cloud-router-2-azure-connection/terraform.tfvars.json"
+          echo $TEST_DATA_PROD_CLOUD_ROUTER_2_PORT_ROUTING_PROTOCOL_CONNECTION >> "./tests/examples-without-external-providers/cloud-router-2-port-routing-protocol-connection/terraform.tfvars.json"
+          echo $TEST_DATA_PROD_CLOUD_ROUTER_2_SERVICE_PROFILE_CONNECTION >> "./examples/cloud-router-2-service-profile-connection/terraform.tfvars.json"
+          echo $TEST_DATA_PROD_CLOUD_ROUTER_2_WAN_CONNECTION >> "./examples/cloud-router-2-wan-connection/terraform.tfvars.json"
+
```
Comment (srushti-patl) [inline .github/workflows/terratests-prod-suite.yaml:59]: nit: remove extra space

**Review by srushti-patl — COMMENTED** [review]
Requested small changes

File: .github/workflows/terratests-prod-suite.yaml (hunk 2 above)
Comment (jkallem-equinix) [reply .github/workflows/terratests-prod-suite.yaml:57]: done

**Review by srushti-patl — APPROVED** [review]
LGTM!

**Pull request merged**